spec/contexts/movies/commands/delete_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+require 'faker'
+
+RSpec.describe Contexts::Movies::Commands::Delete do
+  describe '#call' do
+    before do
+      2.times do
+        create(:movie)
+      end
+    end
```

Comment:
it's better be less generic in the specs 
```suggestion
    before do
      create(:movie)
      create(:movie)
    end
```